navigator/runtime-compose/src/main/java/com/freeletics/mad/navigator/compose/NavHost.kt

```diff
@@ -29,40 +31,34 @@ import com.google.accompanist.navigation.material.rememberBottomSheetNavigator
  * Create a new [androidx.navigation.compose.NavHost] with a [androidx.navigation.NavGraph]
  * containing all given [destinations]. [startRoot] will be used as the start destination
  * of the graph.
- *
- * [destinationCreator] can be passed to add support for custom subclasses of [NavDestination].
  */
 @ExperimentalMaterialNavigationApi
 @Composable
 public fun NavHost(
     startRoot: NavRoot,
     destinations: Set<NavDestination>,
-    destinationCreator: (NavDestination) -> AndroidXNavDestination? = { null },
 ) {
     val bottomSheetNavigator = rememberBottomSheetNavigator()
     val navController = rememberNavController(bottomSheetNavigator)
     val startDestinationId = startRoot.destinationId
-    NavHost(navController, startDestinationId, destinations, destinationCreator)
+    NavHost(navController, startDestinationId, destinations) { null }
```

Comment:
The support for custom destinations only makes sense for the overloads below where a `NavHostController` is passed to us. With the one we are creating here it's not possible to create the custom destinations.